intel_nn_hal/GnaPreparedModel.cpp

```diff
@@ -409,7 +409,7 @@ bool GnaPreparedModel::initialize(const hidl_vec<hidl_handle>& modelCache, const
         if (lstmCount > 3) {
             isEnc1Nw = true;
             modelNameStr = "Encoder1";
-        } else {
+        } else if(lstmCount == 2){
```

Comment:
In case we are following coding conventions, include spacing `if (lstmCount == 2) {`

Reply:
Done